File: src/main/java/com/github/yuppieflu/stats/service/StorageService.java
package com.github.yuppieflu.stats.service;

import com.github.yuppieflu.stats.service.domain.Measurement;
import com.github.yuppieflu.stats.service.domain.Statistic;
import com.github.yuppieflu.stats.service.domain.Status;

public interface StorageService {
    Status addMeasurement(Measurement measurement);
    Statistic getStatistic();
}

File: src/main/kotlin/com/github/yuppieflu/stats/service/StorageService.kt
package com.github.yuppieflu.stats.service

import com.github.yuppieflu.stats.service.domain.Measurement
import com.github.yuppieflu.stats.service.domain.Statistic
import com.github.yuppieflu.stats.service.domain.Status

interface StorageService {
    fun addMeasurement(m: Measurement) : Status
    fun getStatistic(): Statistic
}